helm_deploy/values-uat.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for helm_deploy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: "laa-assess-non-standard-magistrate-fee-app"
+fullnameOverride: "laa-assess-non-standard-magistrate-fee-app"
+
+service:
+  type: ClusterIP
+  port: 80
+  internalPort: 3000
+
+ingress:
+  enabled: true
+  namespace: laa-assess-non-standard-magistrate-fee-uat
+  className: modsec
+  annotations:
+      nginx.ingress.kubernetes.io/enable-modsecurity: "true"
+      nginx.ingress.kubernetes.io/modsecurity-snippet: |
+        SecRuleEngine On
+        SecAction "id:900200,phase:1,nolog,pass,t:none,setvar:tx.allowed_methods=GET HEAD POST OPTIONS PUT PATCH DELETE"
+        SecAction "id:900110,phase:1,nolog,pass,t:none,setvar:tx.inbound_anomaly_score_threshold=6"
+      external-dns.alpha.kubernetes.io/set-identifier: laa-assess-non-standard-magistrate-fee-app-laa-assess-non-standard-magistrate-fee-uat-green
+      external-dns.alpha.kubernetes.io/aws-weight: "100"
+  hosts:
+    - host: nsmassessuat.apps.live.cloud-platform.service.justice.gov.uk
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+          backend:
+          service:
+            name: laa-assess-non-standard-magistrate-fee-uat
+            port:
+              number: 3000
+  tls:
+    - hosts:
+      - nsmassessuat.apps.live.cloud-platform.service.justice.gov.uk
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 500Mi
+  requests:
+    cpu: 10m
+    memory: 500Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+secrets:
+  environment: uat
+
+service_account:
+  name: circleci
```

Comment:
We should not be using the circleci service account as this does not give us the permissions we require and also provides K8s permissions we probably shouldnt be giving to the pod